utils.groovy

```diff
@@ -233,4 +247,97 @@ def getDockerTag() {
   return "${IS_PRO ? 'pro-' : ''}${getDockerBuildOs(env.OS)}-${env.ARCH}-${RSTUDIO_VERSION_FLOWER}"
 }
 
+/**
+  * Rename the file on the filesystem
+  */
+def renameFile(String dir, String match) {
+  def file = sh(script: "basename `ls ${dir}/${match}`", returnStdout: true).trim()
+  def renamedFile = file.replace('-relwithdebinfo', '')
+  sh "mv ${dir}/${file} ${dir}/${renamedFile}"
+  return renamedFile
+}
+
+/**
+  * Rename the file on the filesystem
+  */
+def renameTarFile(String dir) {
+  if ((FLAVOR == "Desktop") || (FLAVOR == "Electron")) {
```

Comment:
Putting this `if` here makes me discomfortable. I would recommend putting it around the calling line instead so that nothing unpredictable happens if we use this elsewhere.

Reply:
I checked the code and I see why this if statement made it in here. We can probably leave it, but it still makes me antsy.

Reply:
Tricky since these were originally linux-only. I can add comments explaining when to use them, but anything in particular that would make you less discomfortable?

Reply:
I'm fine with things being specific to Linux, it's more that we have this conditional here that is opaque unless you open this file and know that it's there, rather than being logic performed directly in the pipeline itself. I think you can leave it since we aren't using it anywhere else.

To make it clearer that they are linux-related you could rename the function to `renameTarFileLinux` or something like that, since `Desktop` `Electron` builds on Windows and Mac as well.